common/src/nttConsts.ts

```diff
@@ -113,14 +113,52 @@ export const NTT_TOKENS: NTTContract = {
   Devnet: {},
 };
 
+export type NTTChain =
+  | 'Ethereum'
+  | 'Fantom'
+  | 'Solana'
+  | 'Arbitrum'
+  | 'Optimism'
+  | 'Base'
+  | 'Sepolia'
+  | 'Holesky'
+  | 'ArbitrumSepolia'
+  | 'BaseSepolia'
+  | 'OptimismSepolia';
+
+const nttChains = [
+  'Ethereum',
+  'Fantom',
+  'Solana',
+  'Arbitrum',
+  'Optimism',
+  'Base',
+  'Sepolia',
+  'ArbitrumSepolia',
+  'BaseSepolia',
+  'OptimismSepolia',
+];
```

Comment:
If you use `as const` at the end here you can simplify the type definitions.
```suggestion
] as const;

export type NTTChain = (typeof nttChains)[number];

export type NTTEvmChain = NTTChain & EvmChains;
```

Reply:
Thanks for the code.  Changed.